src/app/catalog/components/category-tab/category-tab.component.html

```diff
@@ -0,0 +1,2 @@
+<button class="category-equipment-button" routerLink="{{ '/catalog' | categoryPath: item?.id }}"
```

Comment:
let's change the tag to a. Since it is a navigation stuf.

Reply:
I think the pipe should take item and return string and '/catalog' should be defined in the pipe.

Reply:
done